src/App.js

```diff
@@ -19,34 +17,31 @@ const App = () => {
   return (
     <Router>
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1>TCL-16</h1>
-          <p>
-            Welcome to TCL-16! This app component is a good starting place for
-            the Smart Shopping List!
-          </p>
-          <h2>Week 1</h2>
-          <MenuLink activeWhenExact={true} pathTo="/" label="List View" />
-          <MenuLink pathTo="/AddItem" label="Add Item" />
+        {localStorage.getItem('token') ? (
+          <div>
+            <MenuLink
+              activeWhenExact={true}
+              pathTo="/ListView"
+              label="List View"
+            />
+            <MenuLink pathTo="/AddItem" label="Add Item" />
 
-          <Switch>
-            <Route exact path="/">
-              {localStorage.getItem('token') ? (
-                <Redirect to="/ListView" />
-              ) : (
-                <Home />
-              )}
-            </Route>
-            <Route exact path="/ListView">
-              <ListView />
-            </Route>
-            <Route exact path="/AddItem">
-              <AddItem />
-            </Route>
-            <Route component={NotFound} />
-          </Switch>
-        </header>
+            <Switch>
+              <Route exact path="/">
+                <ListView />
+              </Route>
+              <Route exact path="/ListView">
+                <ListView />
+              </Route>
+              <Route exact path="/AddItem">
+                <AddItem />
+              </Route>
+              <Route component={NotFound} />
+            </Switch>
+          </div>
+        ) : (
+          <Home />
```

Comment:
Now no UI except the `Home` component (can likely benefit from a rename) will show if a user doesn't have a token.